ping-protect/src/main/java/org/forgerock/android/auth/PingOneProtectInitializeCallback.kt

```diff
@@ -19,12 +19,8 @@ private val TAG = PingOneProtectInitializeCallback::class.java.simpleName
 /**
  * Callback to initialize the ping one protect
  */
-open class PingOneProtectInitializeCallback : AbstractCallback {
-    @Keep
-    constructor(jsonObject: JSONObject, index: Int) : super(jsonObject, index)
-
-    @Keep
-    constructor() : super()
```

Comment:
i am not sure this will fail , if we minify the project, we may need both constructors, can we just do this PingOneProtectInitializeCallback: AbstractProtectCallback